frontend/attestations/parse.go

```diff
@@ -0,0 +1,71 @@
+package attestations
+
+import (
+	"encoding/csv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	KeyTypeSbom       = "sbom"
+	KeyTypeProvenance = "provenance"
+)
+
+const (
+	defaultSBOMGenerator = "jedevc/buildkit-syft-scanner:latest"
```

Comment:
For safety, this should have a digest. Also a comment that we need to replace this before the release would be good.